builder/remotecontext/urlutil/urlutil.go

```diff
@@ -34,6 +34,7 @@ func IsURL(str string) bool {
 //   - http://(.*).git(?:#.+)?$   same, but non-TLS
 //   - git://(.*)                 URLs using git:// scheme
 //   - git@(.*)
+//   - ssh://git@(.*)
```

Comment:
Why limit the username to "git" ?

Reply:
I dropped the git username. Since the [user name is optional](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol) in the protocol, just reducing it to "ssh://" seems appropriate.